model_analyzer/plots/plot_manager.py

```diff
@@ -53,6 +53,38 @@ def __init__(self, config, result_manager):
         self._simple_plots = defaultdict(list)
         self._detailed_plots = {}
 
+    def to_dict(self):
```

Comment:
I prefer that these aren't in the production code. This code is only for testing and should be able to be hid in the test code

Reply:
I've moved this into the test class.